.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "parser": "babel-eslint",
```

Comment:
This isn't necessary if your project doesn't use any of the fanciest language features (here the package.json says `es2015`), but now that it's there no point in removing it.